### ISSUE TITLE
[BUG] Standalone keeps crashing after changed Buffer Size

### ISSUE THREAD

**Issue by PalpoKelpo** [sdatkinson/NeuralAmpModelerPlugin#424]
**To Reproduce**
1. Open NeuralAmpModeler
2. Click on File, then Preferences
3. Change buffer size

**Expected behavior**
To be able to change the Buffer size so the Program stops crashing

**Screenshots**

**Desktop (please complete the following information):**
 - OS Windows 10
 - Plugin version v.0.7.7
 - Standalone
 - Audio interface Universal Audio Volt 2
 - Sample rate [e.g. 44.1 kHz]
 - Buffer size 96 (can't change it because it crashes)

**Additional context**
Yesterday I was messing around with Buffer Sizes. After changing it to 96 I heard a really weird sound and the Program crashed. 
Now everytime I open it or try to change any setting it keeps on crashing.
I've uninstalled and reinstalled it many times but nothing seemed to work. 

I've explained the error in the Neural Amp Modeler Facebook Group about 10 min ago and uploaded a Video of it happening. 

This is the Link:
https://www.facebook.com/groups/neuralampmodeler/posts/7014242751957821/?comment_id=7014277945287635&notif_id=1704796619308776&notif_t=group_comment

**Comment by sdatkinson** [comment]
You said you picked a buffer size of 96...Am I correct that your interface doesn't support 96-sample buffers?

**Comment by PalpoKelpo** [comment]
Correct. I tried out different buffer sizes because I pretty much don't know anything about this kind of stuff! 
The Problem was that I wasn't even able to change back the buffer size, as it was always crashing. A member of the Facebook Group pointed out though, that I could change the buffer size inside the .ini File inside the N.A.M. folder. I did exactly that and it solved the Problem! 

The people over at the Facebook Group still advised me to post this as a BUG in here, since I wasn't able to change any setting inside the standalone program without it crashing.

**Comment by sdatkinson** [comment]
Ok, good to know.

I'm going to mark this as an upstream bug since the iPlug2 framework is the part of the code that handles that dialog box; the right place for a fix would be there, not in this repo (and by extension, once it's fixed there this plugin (along with others who have made plugins on iPlug2) will inherit it 🙂 ).

If you wanted to check [the Issues there](https://github.com/iPlug2/iPlug2/issues) and see whether this is a known bug you might consider sharing this bug there so that it's in front of the right eyes 👍🏻 .

**Comment by apw100** [comment]
Since Neural Amp Modeler crashes when you try to change the settings, you can do this manually.

1. Open a File Explorer window, open the View dropdown menu and make sure "Hidden Items" is checked.
2. Then go to the following path "C:\Users\XXXX\AppData\Local\NeuralAmpModeler" (replace "XXXX" with your user account name).
3. There should be a single file named "settings". Right click on this, select "Open With" and then select "Notepad"
4. Now edit the settings and save the file.
5. Don't forget to uncheck the "Hidden Items" box in your File Explorer view panel when you're done.
![NeuroFix](https://github.com/sdatkinson/NeuralAmpModelerPlugin/assets/12414945/8ef57155-0a77-475b-9903-1422c5c8082a)

**Comment by sdatkinson** [comment]
Thanks @apw100.

@ghost did this fix your problem?

**Comment by xenoxcosmos** [comment]
IT WORKED! I HAD THE SAME PROBLEM, AND THIS FIXED IT! Thanks!!!
